includes/embed.php

```diff
@@ -145,9 +145,12 @@ public static function get_embed_url( $video_url, array $embed_url_params = [],
 	 * @return string The embed HTML.
 	 */
 	public static function get_embed_html( $video_url, array $embed_url_params = [], array $options = [], array $frame_attributes = [] ) {
+		$video_properties = self::get_video_properties( $video_url );
+
 		$default_frame_attributes = [
 			'class' => 'elementor-video-iframe',
 			'allowfullscreen',
+			'title' => $video_properties['provider'] . ' video iFrame',
```

Comment:
This should be wrapped as a translateable string, ex:
```suggestion
			'title' => sprintf( __( '%s Video Player', 'elementor' ), $video_properties['provider'] ),
```
also not sure about the string it self, how about the actuall video title?

@shipley-dcc 
@shilo-ey

Reply:
As far as I can tell, we don't have the video title available to include here so this is the best information we can provide to the user currently.